fleetspeak/src/e2etesting/e2etest.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 cd $(dirname $0)/../../../
 python3 -m venv /tmp/.venv/FLEETSPEAK
-/tmp/.venv/FLEETSPEAK/bin/pip install absl-py protobuf fleetspeak > /dev/null
```

Comment:
Now I'm not sure, but do we still need fleetspeak? I guess it should be a dependency as well in setup and not needed here.